File: app/src/main/java/com/wilinz/yuetingmusic/util/TimeUtil.java
package com.wilinz.yuetingmusic.util;

public class TimeUtil {
    public static String format(long curPosition) {
        long cm = curPosition / 1000 / 60;
        long cs = curPosition / 1000 % 60;
        StringBuilder builder = new StringBuilder();
        return builder.append(cm / 10).append(cm % 10).append(":")
                .append(cs / 10).append(cs % 10).toString();
    }
}

File: app/src/main/java/com/wilinz/yuetingmusic/util/TimeUtil.kt
package com.wilinz.yuetingmusic.util

object TimeUtil {
    @JvmStatic
    fun format(curPosition: Long): String {
        val cm = curPosition / 1000 / 60
        val cs = curPosition / 1000 % 60
        val builder = StringBuilder()
        return builder.append(cm / 10).append(cm % 10).append(":")
            .append(cs / 10).append(cs % 10).toString()
    }
}